galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import serializers
+from pulp_ansible.app.models import CollectionRemote
+from pulp_ansible.app.viewsets import CollectionRemoteSerializer
+from galaxy_ng.app.constants import COMMUNITY_DOMAINS
+
+
+class SyncConfigSerializer(CollectionRemoteSerializer):
+    created_at = serializers.DateTimeField(source='pulp_created', required=False)
+    updated_at = serializers.DateTimeField(source='pulp_last_updated', required=False)
+    policy = serializers.ChoiceField(
+        choices=CollectionRemote.POLICY_CHOICES,
```

Comment:
This will allow policies other than immediate to be used, but since our convo @rochacbruno I've learned that pulp-ansible cannot support this. We will need to resolve this issue first https://pulp.plan.io/issues/7470

As such I think this specific aspect needs to be reverted (and just use the version defined on the parent class `CollectionRemoteSerializer`.

@newswangerd fyi also ^

Reply:
Ok  @bmbouter thanks for the info, I will remove that by now and keep with the super class implementation.